notes.md

```diff
@@ -0,0 +1,124 @@
+## other notes.
+
+### Terms
+
+  - Price per share approx. `$54`
+  - Nature of securities `Equity`
+  - Closing Date `May 30 (Ideally)`
+  - Amount of securities offered `Approx 55,000`
+
+### Implied Terms
+  - Statement as to the number of days left in the campaign `probably 30 but TBD`
+  - Offering goal `$2.5M`
+
+### Non-terms
+
+* didn't come across as super relevant right now. Closing soon can appear on May 23 perhaps, one week before close.
+
+### Releasing of funds and delayed founder liquidity-
+
+  - both of these concerns are alleviated by the corporate structure.
+
+      1. Invested funds must be used for intended purposes or shareholders can sue us, thereby alleviating the risk of immediately releasing funds.
+      2. We have delayed liquidity in the sense that the company won't own a `premine`, so if we don't continue to perform valuable work for the network, people won't agree to pay our fee on block rewards and fork.
+
+  - Authorized, but unissued shares can be held in an address owned by the company.
+  - We do have to differentiate between never issued and repurchased tokens (treasury shares), however.
+
+      Perhaps just another multisig wallet.
+
+  - Price per share is denominated in USD.
+  - CO2KN, INC will announce its own exchange rate at some point close to the auction.
+  - So, no need for an oracle. This is optimal because the IRS sees each transaction of ETH as a sale. So it's not viewed as one transaction of ETH for Equity, it's seen as USD for equity and then cash for ETH.
+  - Nature of securities takes the following values: `equity`, `debt`, `convertible`, etc.
+  - 'Closing soon' can probably be posted a week before close.
+
+## also need to record
+
+* Name of the shareholder;
+* Complete mailing address of the stock shareholder including contact number;
+* Stock certificate number;
+* The total number of shares outstanding;
+* The date the shares were purchased;
+* How much the person paid per share (consideration/monetary value);
+* The class of shares;
+* Why the shares were transferred.
+
+### implementation
+
+* The `shareholder name`, their `mailing address`, and `contact number` MUST all be stored in an off-chain database and the content hashed, and the `hash` associated with the address.
+* The total number of shares outstanding is the difference between a cap and `token.totalSupply()` and is rightly a function of the `CrowdsaleContract`.
+* The date the share was purchased is a property of the block the purchase was processed in so there is no need to store it in the token. However the user database might wish to record that information as well for convenience.
+* The price the user paid is a function of the ETH price per Token (which can change during the crowdsale) and ETH price at that point (which can also be set in the `CrowdsaleContract`) and the number of ETH spent, which the `CrowdsaleContract` also knows and which is returned in the transaction receipt.
```

Comment:
After the crowdsale is complete, the eth is in escrow and the price paid is logged, we are then able to 'sell eth' at the ratio we originally set in the terms, the terms of which can be contigent on the price at end of sale, the smart-contract would then return the difference in eth to that the buyer .  Am I thinking about this correctly?

Reply:
OR: we issue Tokens to the spender directly with no escrow. If the price of ETH changes substantially between ETH being sent to the Crowdsale  Contract and being converted to USD, then that's not something the spender should care about.  If it drops then C4Coin loses out a bit, if it climbs, then C4Coin gains a bit.

Reply:
@tavakyan, @davesag has the right idea here, but for the purposes of clarification, the price the company sent for ETH is purely for simplicity and tax purposes. Much easier to tell people we are 1) selling shares for $x and 2) we are 'purchasing' ETH at $y so that we can easily calculate a constant x/y=ETH/share.

Thus, CO2KN could end up with a mix of USD and ETH. That's totally fine, because we will probably spend some ETH and if we can't spend it we will sell it when we need to. The fact that we are "buying" it at some price is solely to make the calculation of the exchange rate and tax compliance far, far easier.

Reply:
@tavakyan and, as above, no need for compliance